index.js

```diff
@@ -108,7 +108,30 @@ function resizeImage (filePath, requestData, data, options, cb) {
                 var width, height;
                 var size = metadata.size;
 
-                if (options.scaleUp || (!options.scaleUp && ( size.width > requestData.width || size.height > requestData.height))) {
+
+                // This lets us use just a width or height if desired
+                if("height" in requestData){
+                    requestData.height = parseInt(requestData.height)
+                }
+
+                if("width" in requestData){
+                    requestData.width = parseInt(requestData.width)
+                }
+
+                if(typeof(requestData.height) !== "undefined" && typeof(requestData.width) === "undefined" ){
```

Comment:
Pedantry, I know, but would you mind putting `typeof(requestData.height)` into the preferred format of `typeof requestData.height` as it's an operator and not a function? Also, you probably should check that they contain numbers instead and reverse the operands as that is the only accepted type for height/width.